contract_address.json

```diff
@@ -0,0 +1,135 @@
+[
+    {
+        "Contract Name": "Zone Contract",
+        "Testnet Address": "0x8831867E347AB87FA30199C5B695F0A31604Bb52",
+        "Mainnet Address": "0x00338b92Bec262078B3e49BF12bbEA058916BF91",
+        "Contract Code Link": "",
+        "Tags": ["traders"]
+    },
+    {
+        "Contract Name": "Zone Signer",
+        "Testnet Address": "0x539E1Aa6f2063D8674088fBb0b1aE247Ba1B096b",
+        "Mainnet Address": "0x71f83d1616d1E020053d5d9Ba5CBd385693C2955",
+        "Contract Code Link": "",
+        "Tags": ["traders"]
+    },
+    {
+        "Contract Name": "Read Only Order Validator",
```

Comment:
There's a few auxiliary contracts that do not have public facing/orderbook facing interfaces, and are only used by contracts through internal transactions, not sure how useful it is to make these public. Thoughts @immutable/traders ?